Gemfile.lock

```diff
@@ -315,76 +316,78 @@ GEM
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
```

Comment:
### [connection_pool](https://github.com/mperham/connection_pool)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.2.1 → 2.4.1

[_[change-log](https://github.com/mperham/connection_pool/blob/main/Changes.md), ~~source-code~~, [gem-diff](https://my.diffend.io/gems/connection_pool/2.2.1/2.4.1)_]





<details>
<summary>Commits</summary>

A change of **85** commits. See the full changes on [the compare page](https://github.com/mperham/connection_pool/compare/v2.2.1...v2.4.1).

These are the first 10 commits:
- (1f6109d) [update rubies](https://github.com/mperham/connection_pool/commit/1f6109d821f9e3bca7345c30857d42b41818c3a3)
- (7317ac8) [Add 2.4.1 to travis rubies](https://github.com/mperham/connection_pool/commit/7317ac897fa9b98a1789176c2789757ae5d2e721)
- (7413fbf) [optimize with (15% gain)](https://github.com/mperham/connection_pool/commit/7413fbf13f1475573fc4aa469c8e01246f0e2812)
- (8ac6a0c) [Make size/available stats available for a connection pool.](https://github.com/mperham/connection_pool/commit/8ac6a0c78f216066dd04870816d4a16d38d4d37c)
- (cd6f0e0) [Use separate methods for stats.](https://github.com/mperham/connection_pool/commit/cd6f0e0bfba923a994ef6e807b295fe27a5377f7)
- (b4ce270) [Massage method naming, wrapper support, #97](https://github.com/mperham/connection_pool/commit/b4ce270ef1e30c5660b0ae28320809efd7cad160)
- (6e86113) [add changelog, #97](https://github.com/mperham/connection_pool/commit/6e86113cd85f7aba75176717a2aec0ec8f9d1c28)
- (6dea066) [bump](https://github.com/mperham/connection_pool/commit/6dea06663c80cf50cd6b5a410376943d328b4f1e)
- (08fb434) [[CI] Test against Ruby 2.5](https://github.com/mperham/connection_pool/commit/08fb4343b28da3005a27a3e5887f438a5ff6861f)
- (b00f063) [cleanup](https://github.com/mperham/connection_pool/commit/b00f06350d30430284d754a55d25b902bae097bf)


</details>